src/ccall.cpp

```diff
@@ -1490,13 +1491,11 @@ static jl_cgval_t emit_ccall(jl_codectx_t &ctx, jl_value_t **args, size_t nargs)
     };
 #define is_libjulia_func(name) _is_libjulia_func((uintptr_t)&(name), #name)
 
+    static auto ptls_getter = &jl_get_ptls_states;
```

Comment:
Why the change? This completely defeated the purpose of the old code, which is clearly stated in the comment..............

Reply:
It's because I have been dealing with the dynamic (runtime) linker too much, so when it said "can cause linker issues" I naively thought that meant at runtime when the symbol actually gets used.  I didn't realize this caused issues at compile-time link.  :P

Reply:
You are welcome to change the comment in whichever way you want so that anyone that have read that comment won't change this code without testing the compiler configure combination again.....